src/main/java/uk/gov/hmcts/reform/civil/service/PaymentServiceRequest.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.civil.service;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.payments.client.models.FeeDto;
+
+@Data
+@Builder
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PaymentServiceRequest {
+
+    @JsonProperty("call_back_url")
+    private String callBackUrl;
+    @JsonProperty("case_payment_request")
+    private CasePaymentRequestDto casePaymentRequest;
+    @JsonProperty("case_reference")
+    private String caseReference;
+    @JsonProperty("ccd_case_number")
+    private String ccdCaseNumber;
+    @JsonProperty("fees")
+    private FeeDto[] fees;
+    @Builder.Default
+    @JsonProperty("hmcts_org_id")
+    private String hmctsOrgId = "ABA5";
```

Comment:
hardcoded from private law not applicable to us

Reply:
Removed hardcoding and fetching value from paymentConfiguration class siteId